README.md

```diff
@@ -217,11 +217,22 @@ extension has the following configuration options:
   APIs. This is the equivalent to using `--unstable` on the command line.
   _boolean, default `false`_
 
+## Compatibility
+
+To see which versions of the Deno CLI are compatible with which versions of this
+extension, consult the following table.
+
+| vscode-deno  | Deno CLI       |
+| ------------ | -------------- |
+| TODO onwards | 1.37.0 onwards |
```

Comment:
I'll fill this when cutting the release. We know it's gonna be 3.28.0 this time but for the sake of procedure..